codegenerator/cli/src/cli_args/interactive_init/mod.rs

```diff
@@ -138,22 +139,7 @@ pub async fn prompt_missing_init_args(
         }
     };
 
-    let language = match init_args.language {
-        Some(args_language) => args_language,
-        None => {
-            let options = Language::iter()
-                .map(|language| language.to_string())
-                .collect::<Vec<String>>();
-
-            let input_language = Select::new("Which language would you like to use?", options)
-                .with_starting_cursor(1)
-                .prompt()
-                .context("prompting user to select language")?;
-
-            Language::from_str(&input_language)
-                .context("parsing user input for language selection")?
-        }
-    };
+    let language = Language::TypeScript;
```

Comment:
I'd do it this way. Also need to perform this check only for contract import flow. Since templates are TS only.
```suggestion
    let language = match init_args.language {
        Some(args_language) => args_language,
        None => Language::TypeScript
    };
```